packages/web/src/pages/trending-page/components/RewardsBanner.tsx

```diff
@@ -87,32 +88,22 @@ const RewardsBanner = ({ bannerType }: RewardsBannerProps) => {
     >
       <Flex
         direction={isMobile ? 'column' : 'row'}
-        css={{
-          marginRight: isMobile ? 'unset' : 'auto',
-          marginBottom: isMobile ? spacing.s : 0,
-          width: isMobile ? '100%' : 'auto'
-        }}
+        w='100%'
         alignItems={isMobile ? 'flex-start' : 'center'}
         gap={isMobile ? undefined : 'l'}
       >
-        <Flex
-          alignItems='center'
-          css={{
-            marginBottom: isMobile ? spacing.xs : 0
-          }}
-          gap='s'
-        >
+        <Flex alignItems='center' mb={isMobile ? 'xs' : undefined} gap='s'>
           <IconCrown size='l' />
-          <Text variant='title' color='staticWhite'>
+          <Text variant='title' size='l' color='staticWhite'>
             {messageMap[bannerType].title}
           </Text>
         </Flex>
         <Text
           variant='body'
           size='l'
-          strength={isMobile ? 'strong' : 'weak'}
+          strength='strong'
           css={{
-            marginLeft: isMobile ? 0 : spacing.s,
+            opacity: 0.8,
             marginTop: isMobile ? spacing.xs : 0,
```

Comment:
i guess still wondering if we can drop these styles? ideally should all just be flex centered right?